docs/docsite/rst/guide_deps.rst

```diff
@@ -66,9 +66,7 @@ they can be validated selectively by doing:
 .. code-block:: python
 
     deps.validate(module, "foo")       # only validates the "foo" dependency
-
     deps.validate(module, "doe:bar")   # only validates the "doe" and "bar" dependencies
-
     deps.validate(module, "-doe:bar")  # validates all dependencies except "doe" and "bar"
```

Comment:
Why remove the newlines? I think it's nicer to read with them (less crowded :) )

(I would maybe even move the comment in the line before the statement, and still have a newline - uses more vertical space, but less horizontal space.)

Reply:
I guess that falls under "personal preferences". The form you described seems too "stacked" to me. I am happy to return the newlines there though, sounds like middle ground to me.

Reply:
Of course is it :)

On the other hand using less horizontal space in code examples is useful for narrow screens, like when looking at the docs page on your mobile phone or on a tablet in portrait mode. Having to scroll to the right to be able to fully read the comments isn't great either :)